src/modules/LanItems.rb

```diff
@@ -2341,15 +2341,19 @@ def createUdevFromIfaceName(interfaces)
         "bus" => "KERNELS"
       }
 
+      # rubocop:disable Next
+      # the check is disabled bcs the code uses capture groups. Rewriting
+      # the code would require some tricks to access these groups
       interfaces.each do |interface|
-        next if /.*-(?<attr>id|bus)-(?<value>.*)/ !~ interface["device"]
```

Comment:
This is interesting. [The documentation](http://ruby-doc.org/core-2.2.0/Regexp.html#class-Regexp-label-Capturing) says:

> When named capture groups are used with a literal regexp on the left-hand side of an expression and the `=~` operator, the captured text is also assigned to local variables with corresponding names.

So you could keep the `next` if you used `=~` instead of `!~`:

```rb
next unless /.*-(?<attr>id|bus)-(?<value>.*)/ =~ interface["device"]
```